RHEL/7/input/xccdf/services/docker.xml

```diff
@@ -0,0 +1,90 @@
+<Group id="docker">
+<title>Docker Daemons</title>
+<description>The cron and at services are used to allow commands to
+be executed at a later time. The cron service is required by almost
+all systems to perform necessary maintenance tasks, while at may or
+may not be required on a given system. Both daemons should be
+configured defensively.</description>
+
+<Rule id="service_crond_enabled" severity="medium">
```

Comment:
There already exists ```service_crond_enabled``` rule in ```RHEL/7/input/xccdf/services/cron.xml```. What this means being if the Docker profile should utilized the ```service_crond_enabled``` rule, it's enough to reference that rule in the Docker profile already.